core/dbt/exceptions.py

```diff
@@ -1000,6 +1000,12 @@ def warn(msg, node=None):
     return ""
 
 
+# TODO: remove this later as Sung created this to avoid errors with running dbt-snowflake==1.3.0
+def warn_or_error(msg, node=None):
```

Comment:
Is there a reason this function was removed from the `main` branch while `1.3.latest` still has it? https://github.com/dbt-labs/dbt-core/blob/1.3.latest/core/dbt/exceptions.py#L1090

Reply:
Ah, this was related to work that Emily has been doing on more structured logging/exceptions. Just opened a ticket for it: https://github.com/dbt-labs/dbt-core/issues/6297

Reply:
Looks like the selector error message was fixed, but is there a reason `warn_or_error` doesn't exist in the `main` branch but in `1.3.latest`? @jtcohen6

Reply:
@sungchun12 It still exists, it just moved:
```
- from dbt.exceptions import warn_or_error
+ from dbt.events.functions import warn_or_error
```

Reply:
Ahhhh I know what happened, I needed to update my snowflake adapter to the main branch vs. relying on `1.3.latest` Thanks Jerco! I'm unblocked!